packages/applications/legacy/src/controllers/project/getProjectPage/_utils/getReprésentantLégal.ts

```diff
@@ -38,17 +39,20 @@ export const getReprésentantLégal: GetReprésentantLégal = async (identifiant
       });
 
     if (Option.isSome(représentantLégal)) {
+      const featureActivated = isDemandeChangementReprésentantLégalEnabled();
```

Comment:
De quelle feature parles tu ? J'aurais gardé un truc explicite

```suggestion
      const featureDemandeChangementReprésentantLégalEnabled = isDemandeChangementReprésentantLégalEnabled();
```